src/routes/issue-trigger-rule/get.ts

```diff
@@ -0,0 +1,39 @@
+import Joi from 'joi';
+import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+import { IssueTriggerRule } from '../../models/issue-trigger-rule';
+
+type HashId = string;
+type Request = {
+  params: {
+    hashId: HashId
+  }
+};
+type EffectiveRequest = Request;
+
+type EnvironmentIssueTriggerRule = IssueTriggerRule;
+
+type Response = EnvironmentIssueTriggerRule
+
+const controllerGeneratorOptions: ControllerGeneratorOptionsWithClient = {
+  method: 'get',
+  path: '/:hashId',
+  params: Joi.object().keys({
+    hashId: Joi.string().required().example('xd2rd4'),
+  }),
+  right: { environment: 'READ' },
+  response: Joi.object().keys({
+    clientHashId: Joi.string().example('xd2rd4').required(),
+    deviceTypeHashId: Joi.string().example('xd2rd4'),
+    priorityLevel: Joi.number().integer().valid(1, 2).example('1'),
+    missedReports: Joi.number().integer().required().example('12'),
+    offlineForSeconds: Joi.number().integer().required().example('1800'),
+  }),
+  description: 'Gets all issue trigger rules for an environment',
```

Comment:
I would still prefer the description that we use elsewhere: 'Get a specific xxxxx identified by its hashId'